utils/armClientGenerator/generator.ts

```diff
@@ -16,14 +16,14 @@ Results of this file should be checked into the repo.
 */
 
 // CHANGE THESE VALUES TO GENERATE NEW CLIENTS
-const version = "2023-04-15";
+const version = "2023-09-15-preview";
```

Comment:
Any reason to use the preview and not wait for the stable API? There's already been a change to the preview release.

Reply:
This has been discussed with Armando and the preview API is needed so that we can get the status for priority feature is enabled or not